internal/xds/rbac/converter.go

```diff
@@ -24,13 +24,17 @@ import (
 	v1xdsudpatypepb "github.com/cncf/xds/go/udpa/type/v1"
 	v3xdsxdstypepb "github.com/cncf/xds/go/xds/type/v3"
 	v3rbacpb "github.com/envoyproxy/go-control-plane/envoy/config/rbac/v3"
+	v3auditloggersstreampb "github.com/envoyproxy/go-control-plane/envoy/extensions/rbac/audit_loggers/stream/v3"
 	"google.golang.org/grpc/authz/audit"
+	"google.golang.org/grpc/authz/audit/stdout"
+	"google.golang.org/protobuf/encoding/protojson"
 	"google.golang.org/protobuf/types/known/anypb"
 	"google.golang.org/protobuf/types/known/structpb"
 )
 
 const udpaTypedStuctType = "type.googleapis.com/udpa.type.v1.TypedStruct"
 const xdsTypedStuctType = "type.googleapis.com/xds.type.v3.TypedStruct"
+const stdoutType = "type.googleapis.com/envoy.extensions.rbac.audit_loggers.stream.v3.StdoutAuditLog"
```

Comment:
```suggestion
const (
	udpaTypedStructType = "type.googleapis.com/udpa.type.v1.TypedStruct"
	xdsTypedStructType = "type.googleapis.com/xds.type.v3.TypedStruct"
	stdoutType = "type.googleapis.com/envoy.extensions.rbac.audit_loggers.stream.v3.StdoutAuditLog"
)
```
pls. While you are at there, might be helpful to fix typos

Reply:
I'm not immediately seeing any typos, where are they?

Reply:
`udpaTypedStuctType` -> `udpaTypedStructType`

Reply:
Ah thank you! Fixed